include/dem/dem.h

```diff
@@ -142,6 +138,12 @@ class DEMSolver
 #  endif
 
 
+  /**
+   * @brief Establish if this is a contact detection step using the adequate contact search method
```

Comment:
I haven't done Doxygen so far, but as this is a return function, shouldn't it have a return instead (or with) a brief? Same question for the member is_load_balance_step().